pyQuARC/schemas/rule_mapping.json

```
@@ -4591,5 +4599,53 @@
         },
         "severity": "info",
         "check_id": "date_compare"
+    },
+    "validate_granule_data_format_against_collection_check": {
+        "rule_name": "Validate Granule Data Format Against Collection Check",
+        "fields_to_apply": {
+            "umm-g": [
+                {
+                    "fields": [
+                        "DataGranule/ArchiveAndDistributionInformation/Format",
+                        "CollectionReference/ShortName",
+                        "CollectionReference/Version",
+                        "CollectionReference/EntryTitle"
+                    ]
+                }
+            ],
+            "echo-g": [
+                {
+                    "fields": [
+                        "Granule/DataFormat",
+                        "Granule/ShortName",
+                        "Granule/Version",
+                        "Granule/Collection/DataSetId"
+                    ]
+                }
+            ]
+        },
+        "severity": "error",
+        "check_id": "validate_granule_data_format_against_collection_check"
+    },
+    "granule_data_format_presence_check": {
+        "rule_name": "Granule Data Format Presence Check",
+        "fields_to_apply": {
+            "umm-g": [
+                {
+                    "fields": [
+                        "DataGranule/ArchiveAndDistributionInformation/Format"
+                    ]
+                }
+            ],
+            "echo-g": [
+                {
+                    "fields": [
+                        "Granule/DataFormat"
+                    ]
+                }
+            ]
+        },
+        "severity": "warning",
+        "check_id": "granule_data_format_presence_check"
     }
-}
+}
```

```suggestion
}

```
I think it's good practice to add a new line to the end of a file